### PR TITLE
UI: Adapt Logging Screen for Debug Log (EXPOSUREAPP-4451)

### PR DESCRIPTION
**Pull request by axelherbstreith**
PR updates the main debug log screen with the latest designs. 

To tests the changes, navigate to "App-Information" and select entry "Fehlerberichte".

Please verify if:

- Design is as expected on your device
-  Start Button is clickable -> Color is changed to red and two additional buttons show up
- Link to FAQ is clickable
- Navigation to Privacy Fragment is working

### CODE REVIEW

File: Corona-Warn-App/src/main/res/color/button_debug_log.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/color/button_debug_log.xml:2]: "button_debug_log" is a bit unspecific :wink:

File: Corona-Warn-App/src/main/res/layout/bugreporting_debuglog_fragment.xml [hunk 2]
```diff
@@ -1,114 +1,258 @@
...
+                    android:focusable="true"
+                    android:layout_marginHorizontal="24dp"
+                    android:paddingVertical="@dimen/spacing_tiny"
+                    android:text="@string/contact_diary_onboarding_legal_information"
                     app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toEndOf="@id/share_recording"
-                    app:layout_constraintTop_toBottomOf="@id/debuglog_status_secondary" />
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/debug_log_first_divider"
+                    tools:text="@string/contact_diary_onboarding_legal_information" />
```
[the remark this answers is not in the document]
Comment (d4rken) [reply Corona-Warn-App/src/main/res/layout/bugreporting_debuglog_fragment.xml:134]: The "send screen" also uses this component so we could already re-use it within this PR.

File: Corona-Warn-App/src/main/res/layout/bugreporting_debuglog_fragment.xml [hunk 3]
```diff
@@ -1,114 +1,258 @@
...
+            android:layout_marginBottom="16dp"
+            android:layout_marginTop="@dimen/spacing_normal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintBottom_toTopOf="@+id/toggle_send_error_log">
+
+
+            <ImageView
+                android:id="@+id/debuglog_activity_indicator"
+                android:layout_width="36dp"
```
[the remark this answers is not in the document]
Comment (d4rken) [reply Corona-Warn-App/src/main/res/layout/bugreporting_debuglog_fragment.xml:180]: Or more padding between top and bottom text.

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -104,6 +104,11 @@
         <item name="android:textColor">@color/button_text_color_emphasized</item>
     </style>
 
+    <style name="debugLogButton" parent="button">
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/values/styles.xml:107]: Same here, `debugLogButton` is very generic. Generic is okay if we want to re-use it in the app, but then maybe name it  something like `buttonBarAlertRed` or so.

**Review by d4rken — APPROVED** [review]
lgtm, minor comments that are not blockers.
Waiting for UA review, then this can be merged from my POV.

**Review by janetback — APPROVED** [review]
UA reviewed

**Pull request merged**